src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,154 @@
+              variant="primary"
+              style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+            >
+              Sign Up
+            </Button>
+          </Link>
+        </div>
+      </div>
+    </Router>
+  );
+};
+
+const NavbarForSmallScreen = () => {
+  const [sidebar, setSidebar] = useState(false);
+
+  const showSidebar = () => setSidebar(!sidebar);
+
+  const boxRef = useRef(null);
+
+  useEffect(() => {
+    const handleClickOutside = (event) => {
+      if (boxRef && boxRef.current.contains(event.target)) {
+        setSidebar(true);
+      } else {
+        setSidebar(false);
+      }
+    };
+
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, []);
+
+  return (
+    <>
+      <div className="navbarMobile">
+        <Link to="#" className="menuBars">
+          <List onClick={showSidebar} />
+        </Link>
+        <Link to="/" className="navbarMobileLogo">
+          LOGO
+        </Link>
+      </div>
+      <nav ref={boxRef} className={sidebar ? "navMenu active" : "navMenu"}>
+        <ul className="navMenuItems">
+          <li className="navbarToggle" onClick={showSidebar}>
+            <Link to="#" className="menuBars">
+              <List />
+            </Link>
+          </li>
+
+          <li className="navLi navLiLoginSign" onClick={showSidebar}>
+            <Link to="/logIn">
+              <Button
+                variant="primary"
+                style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+              >
+                Log In
+              </Button>
+            </Link>
+          </li>
+          <li className="navLi navLiLoginSign" onClick={showSidebar}>
+            <Link to="/signUp">
+              <Button
+                variant="primary"
+                style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+              >
+                Sign Up
+              </Button>
+            </Link>
+          </li>
+
+          {smallScreenNavbarData.map((info, index) => (
+            <li className={info.cName} onClick={showSidebar} key={index}>
+              <Link to={info.target} className={info.cNameLink}>
+                {info.component} &nbsp; {info.name}
+              </Link>
+            </li>
+          ))}
+          <li className="navLi navLiLang">
+            <GeoAlt style={{ color: "black" }} />
+            <DropdownButton id="dropdown-basic-button" title="Language">
+              <Dropdown.Item href="#/action-1" onClick={showSidebar}>
+                TUR
+              </Dropdown.Item>
+              <Dropdown.Item href="#/action-2" onClick={showSidebar}>
+                ENG
+              </Dropdown.Item>
+            </DropdownButton>
+          </li>
+        </ul>
+      </nav>
+    </>
+  );
+};
+
+const Navbar = () => {
+  getWindowDimensions();
+
+  useWindowDimensions();
+
+  const { width } = useWindowDimensions();
```

Comment:
u only need this line of code here please delete line 143 and line 141, and the extra import for the getWindowDimensions, make sure to read the code and understand it really well, getWindowDimensions is a helper function for the useWindowDimensions hook